recipe/meta.yaml

```diff
@@ -23,16 +23,17 @@ requirements:
     - fastapi
     - pluggy >=1.0.0,<2.0.0
     - toml
-    - click
+    - typer
```

Comment:
I thought the cli was only related to the `uvicorn` plugin, not packaged with the bare minimum `fps`. It's not the case anymore?

Reply:
Well, `fps` already had `click` which is related to the CLI, right?